Makefile

```diff
@@ -17,6 +17,9 @@ DEPLOYMENT_NAME=vsecm
 include ./makefiles/VSecMMacOs.mk
 include ./makefiles/VSecMDeploy.mk
 
+## Keygen
+include ./makefiles/VSecMKeyGen.mk
```

Comment:
The entry point to build the "keygen" container image.

We’ll use the keygen to manually create the root key if the user does not want to use auto-generated VSecM keys.